src/ServiceControl.Plugin.Nsb5.SagaAudit/Messages/SagaChangeInitiator.cs

```diff
@@ -7,7 +7,7 @@ class SagaChangeInitiator
         public string InitiatingMessageId { get; set; }
         public string MessageType { get; set; }
         public bool IsSagaTimeoutMessage { get; set; }
-        public DateTime TimeSent { get; set; }
+        public DateTime? TimeSent { get; set; }
```

Comment:
I always return DateTime.MinValue (as opposed to null) - see Line94. So SC won't break. 
I wanted to be able to return a null at some point, when SC can accept a null. 
Does that make sense?